<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>txt2curves</class>
 <widget class="QMainWindow" name="txt2curves">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>txt2curves</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Okily dokily</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input file:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inputButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>open...</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="labelAngle">
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelZ">
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxZ">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxScale">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Split</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>396</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
